src/Classes/ServiceAPI/MyRadio_Show.php

```diff
@@ -586,20 +586,6 @@ public function getGenre()
         return isset($this->genres[0]) ? $this->genres[0] : null;
     }
 
-    public function isCurrentUserAnOwner()
```

Comment:
This appears to be in MyRadio_Show, and it's moved to MyRadio_Creditable? Or is this not a problem?